VoxelSniperCore/src/main/java/com/github/kevindagame/brush/SchematicBrush.kt

```diff
@@ -163,7 +166,7 @@ class SchematicBrush : AbstractBrush() {
                                 v.sendMessage(Messages.SCHEMATIC_LOADED_MULTIPLE.replace("%schematics%", this.schematics.joinToString(", ") { it.name }))
                             }
                         } catch (e: IllegalArgumentException) {
-                            v.sendMessage(Messages.INVALID_BRUSH_PARAM)
+                            v.sendMessage(Messages.FILE_LOAD_FAIL.replace("%exception.getMessage%", e.message ?: "Unknown error"))
```

Comment:
Did you also add this to the messages file? or was it already in there, also maybe use %message% instead of %exception.getMessage%

Reply:
It was already in there